feodo/package.yaml

```diff
@@ -21,23 +21,24 @@ pipelines:
     name: Update Feodo Context
     description: A pipeline that periodically refreshes the Feodo lookup-table context.
     definition: |
-      every 1h
-        from https://feodotracker.abuse.ch/downloads/ipblocklist_aggressive.csv read csv --allow-comments
-        | context update feodo --key=dst_ip --clear
+      // tql2
+      every 1h {
+        load_http "https://feodotracker.abuse.ch/downloads/ipblocklist_aggressive.csv"
+        read_csv allow_comments=true
+      }
+      context::update "feodo", key="dst_ip" --clear
     restart-on-error: 1h
 
 examples:
   - name: Lookup Feodo
     description: Evaluate all stored data with a `src_ip` field against the Feodo IP Blocklist.
     definition: |
-      lookup feodo --field=dst_ip
+      // tql2
+      legacy "lookup feodo --field=dst_ip"
```

Comment:
From here onward: let's remove the `lookup` examples from packages.

Reply:
Why is that? Just because the operator isn't ported yet?

Reply:
First, it's not an easy-to-understand operator.

Second, it provides you everything from the past plus continues live, without terminating. I think completing examples are nicer.

Third, it's not ported, yes. Unlike `chart`, it will take a lot longer. It would be nice to get rid of `legacy` way before that.